eve/workers/end2end/scripts/vault-e2e-test.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+set -exu
+
+DIR=$(dirname $0)
+
+. "$DIR/common.sh"
+
+ZENKO_RESOURCE=${1:-end2end}
+NAMESPACE=${2:-default}
+DEPS_PATH=${3:-'solution/deps.yaml'}
+
+POD_NAME="${ZENKO_RESOURCE}-vault-test"
+
+# set environment vars
+VAULT_ENDPOINT="http://${ZENKO_RESOURCE}-management-vault-iam-admin-api:80"
+VAULT_STS_ENDPOINT="http://${ZENKO_RESOURCE}-connector-vault-sts-api:80"
+VAULT_AUTH_ENDPOINT="http://${ZENKO_RESOURCE}-connector-vault-auth-api:80"
+ADMIN_ACCESS_KEY_ID=$(kubectl get secret $ZENKO_RESOURCE-management-vault-admin-creds.v1 -n $NAMESPACE -o  jsonpath='{.data.accessKey}' | base64 -d)
+ADMIN_SECRET_ACCESS_KEY=$(kubectl get secret $ZENKO_RESOURCE-management-vault-admin-creds.v1 -n $NAMESPACE -o  jsonpath='{.data.secretKey}' | base64 -d)
+MONGODB_TEST_HOST="${ZENKO_RESOURCE}-base-db"
+MONGODB_TEST_PORT="27017"
+MONGODB_TEST_USER=$(kubectl get secret $ZENKO_RESOURCE-base-db-auth -n $NAMESPACE -o jsonpath='{.data.username}' | base64 -d)
+MONGODB_TEST_PASSWORD=$(kubectl get secret $ZENKO_RESOURCE-base-db-auth -n $NAMESPACE -o jsonpath='{.data.password}' | base64 -d)
+MONGODB_TEST_DATABASE=$(kubectl get zenko $ZENKO_RESOURCE -n $NAMESPACE -o jsonpath='{.status.instanceID}')
+REDIS_TEST_HOST="${ZENKO_RESOURCE}-base-cache"
+REDIS_TEST_PORT="6379"
+REDIS_TEST_PASSWORD=$(kubectl get secret $ZENKO_RESOURCE-base-cache-creds.v1 -n $NAMESPACE -o jsonpath='{.data.password}' | base64 -d)
+VAULT_TEST_TAG=$(kubectl get deployment $ZENKO_RESOURCE-connector-vault -n $NAMESPACE -o jsonpath='{.spec.template.spec.containers[0].image}' | rev | cut -d: -f1 | rev)
```

Comment:
It's not easier than what you did here, but I would consider retrieving this tag from the zenkoversion object.
But then it's not one command it's two, because you have to get the zenko object, check the version and then get the zenkoversion object with the correct name that you retrieved previously. So overall you can ignore me, just thought about sharing this one :D